docroot/themes/humsci/humsci_basic/src/scss/components/_global-footer.scss

```diff
@@ -0,0 +1,11 @@
+.hb-global-footer {
```

Comment:
could you write a little comment at the top here about doing some overrides to decanter because of our theme styles? thank you!